### DISCUSSION TITLE
Chat Bubble Typing Animation

### DISCUSSION THREAD

**Discussion by Elod-T** [saadeghi/daisyui#1728] [Ideas / feature requests]
Hi all!
I was using the new chat components the other day and needed to add a typing animation while the other user is typing. I found [this pen](https://codepen.io/bilalck/pen/jdGGNW), which I think looks awesome. I recreated it with tailwind and thought that it would be a cool addition to this library. When you think it's a good idea I can create a pull request.

**Comment by saadeghi** [comment]
Thank you for the idea.  But I think something like this can be used anywhere. It's a loading animation. Can be made with a gif or CSS or SVG. Can be used in chat bubble or anywhere else.

Also, the animation is really opinionated and people may need different animations so I think it should be up to the developer how and where they want to use a loading animation.
